### PR TITLE
Added missing point types for infoCard

### PR DESCRIPTION
**Pull request by CarenRizk**
Realized some points have types that are not checked when adding them to the info card and fixed info card icon colors 
image

### CODE REVIEW

File: app/src/main/java/com/example/concordia_campus_guide/fragments/PathInfoCardFragment.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public void createDirectionImageButton(final LinearLayout layout, final LinearLa
         final ImageButton btn = new ImageButton(getContext());
         btn.setLayoutParams(layoutParams);
         btn.setImageResource(imageId);
-        btn.setColorFilter(R.color.colorAppTheme);
+        btn.setColorFilter(getResources().getColor(R.color.colorAppTheme));
```
Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/fragments/PathInfoCardFragment.java:109]: isn't this deprecated? You can use the getColor from the activity instead.

Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/fragments/PathInfoCardFragment.java:109]:
```suggestion
        btn.setColorFilter(getActivity().getColor(R.color.colorAppTheme));
```

File: app/src/main/java/com/example/concordia_campus_guide/view_models/PathsViewModel.java [hunk 2]
```diff
@@ -143,6 +141,21 @@ private void addIndoorDescriptionToList(WalkingPoint startWalkingPoint, WalkingP
                     addCardToList("Walk towards classroom " + endWalkingPoint.getPlaceCode(), "Classroom");
                 }
                 break;
+            case PointType.WASHROOM:
```
Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/view_models/PathsViewModel.java:144]: Is it normal that these are hardcoded? Does it mean it won't be translated?

File: app/src/main/res/drawable/ic_directions_washroom.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (viveanban) [inline app/src/main/res/drawable/ic_directions_washroom.xml:1]: Can we use a different washroom icon? like a toilet or something more descriptive?

Comment (CarenRizk) [reply app/src/main/res/drawable/ic_directions_washroom.xml:1]: I couldn't find any :/

File: app/src/main/java/com/example/concordia_campus_guide/view_models/PathsViewModel.java (hunk 2 above)
Comment (CarenRizk) [reply app/src/main/java/com/example/concordia_campus_guide/view_models/PathsViewModel.java:144]: That's true! I'll add them to the resources but I need someone to translate them for me :P

Comment (viveanban) [reply app/src/main/java/com/example/concordia_campus_guide/view_models/PathsViewModel.java:144]: Perfect! I'll help you out!

File: app/src/main/res/drawable/ic_directions_washroom.xml (hunk 3 above)
Comment (viveanban) [reply app/src/main/res/drawable/ic_directions_washroom.xml:1]: How about wc ? (through the android clipart thing)

image

Comment (CarenRizk) [reply app/src/main/res/drawable/ic_directions_washroom.xml:1]: Ohhhh didn't see this yeah I'll replace it !

**Pull request merged**